app/adminPage.html

```diff
@@ -40,52 +41,60 @@ <h3 id="modal-title"> </h3>
                         <input type="text" id="email" name="email" placeholder="Type new user's email">
                         <input type="number" id="time" name="time" value="30">
-                    </div>
-                    <div class="container_submit">
-                        <input type="submit" value="Add User" class="primary-button">
+                        <input type="submit" value="Add User" class="btn primary-button">
                     </div>
                 </form>
                 <div id='error' class='title_input'></div>
-                <h4 class="search-heading">Search User</h4>
-                    <input type="text" id="searchInput" placeholder="Type name or email">
-                    <input type="submit" id="searchSubmit" value="Search" class="primary-button">
-                    <input type="submit" id="searchReset" value="Clear Search" class="primary-button">
-                <div class="filterContainer">
-                    <h4 class="search-heading">Filter Results</h4>
-                    <label for="filterScorePercentage" class="labelDropdown">Score percentage:</label>
-                    <select name="filterScorePercentage" id="filterScorePercentage" class="dropdown">
-                        <option value="" disabled selected>Score Percentage</option>
-                        <option value="3">-none-</option>
-                        <option value="1">+70%</option>
-                        <option value="2">-69%</option>
-                    </select>
-                    <label for="startDate">Start:</label>
-                    <input id="startDate" type="date">
-                    <label for="endDate">End:</label>
-                    <input id="endDate" type="date">
-                </div>
-            <div>
-                <h2>Registered Users</h2>
-                <div class="user_list"></div>
-            </div>
-            <div>
+            <div class="score-container">
                 <h2>Scores</h2>
-                <section>
-
-
-                </section>
-                <div class="score_list"></div>
+                <div class="search-filter-container">
+                    <h4 class="search-heading">Search User</h4>
+                    <div class="display-flex">
+                        <input type="text" id="searchInput" placeholder="Type name or email">
+                        <input type="submit" id="searchSubmit" value="Search" class="btn primary-button">
+                        <input type="submit" id="searchReset" value="Clear Search" class="btn primary-button">
+                    </div>
+                        <h4 class="search-heading">Filter Results</h4>
+                        <!--                    <label for="filterScorePercentage" class="labelDropdown">Score percentage:</label>-->
+                        <!--want to delete the label but will wait until peer reviewed-->
```

Comment:
If site works while this is commented then you can delete this

Reply:
ok